README.md

```diff
@@ -45,47 +90,63 @@ Then execute commands:
     ./scripts/feeds install weather-station-gateway
     make menuconfig
 
-in menuconfig please press `/` and type `weather-station-gateway` one occurrence will appear. Mark it with `<*>` and do save of config.
-In terminal type `make` to build openwrt image with this application. After uploading to ci40 edit file located in `/etc/init.d/weather_station_initd` and put proper switch arguments related to clicks which you put into mikroBus port.
+In menuconfig please press `/` and type `weather-station-gateway` one occurrence 
+will appear. Mark it with `<*>` and do save of config.
+In terminal type `make` to build openwrt image with this application.
+After uploading to ci40 edit file located in `/etc/init.d/weather_station_initd` 
+and put proper switch arguments related to clicks which you put into mikroBus port.
+
+---
+
+## Setup For Execution
+
+Lumpy implements an AwaLWM2M Client to communicate the with the Creator Device 
+Server and its required few steps to be done before running the project. (Please, 
```

Comment:
"its required few steps to be done"
Not sure whether this is grammatically proper. Shouldn't it be
"it requires few steps to be done" ?